recipes/sigmut/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+./sigprofiler -h
+cp -r * ${PREFIX}/bin/
```

Comment:
Please just copy `sigprofiler` and `lib_repo`.

Reply:
Greats, thanks a lot @dpryan79 ! Finally, all tests have passed and PR is ready to be merged.